internal/querycoordv2/observers/handoff_observer.go

```diff
@@ -115,6 +136,13 @@ func (ob *HandoffObserver) StartHandoff(collectionIDs ...int64) {
 	}
 }
 
+func (ob *HandoffObserver) getHandoffEventNum() int {
```

Comment:
Any method accessed outside should be public, `getHandoffEventNum` -> `EventNum` or `GetEventNum`, the type name has implied they are handoff event

Reply:
just for ut, no need to expose to outside.